src/org/gk/render/AbstractNodeRenderer.java

```diff
@@ -51,8 +54,13 @@ public void render(Graphics g) {
     }
 
     protected abstract void renderShapes(Graphics g);
-    
-    protected void renderDrug(Graphics g) {
+
+    protected void renderDrugLabel(Graphics g) {
+    	// Default to rectangle.
+		renderDrugLabel(g, DRUG_SHAPE.RECTANGLE);
```

Comment:
As I said, please don't use DRUG_SHAPE. We may use different shapes for chemicals in the future. Therefore, this is not robust enough, and not flexible too.

Reply:
Ah, my mistake. I missed that recommendation. I will change to something more robust.

Reply:
Renamed to `RENDER_SHAPE` in [0ae7e0](https://github.com/reactome/CuratorTool/pull/23/commits/0ae7e0e6a2431ef5d9c1ca75f94794f93380743b).